cmd/xl-v1-multipart.go

```diff
@@ -1077,13 +1077,51 @@ func (xl xlObjects) CompleteMultipartUpload(bucket string, object string, upload
 	return objInfo, nil
 }
 
+// Wrapper which removes all the uploaded parts.
+func (xl xlObjects) cleanupUploadedParts(bucket, object, uploadID string, storageDisks []StorageAPI) error {
+	var errs = make([]error, len(storageDisks))
+	var wg = &sync.WaitGroup{}
+
+	// Construct uploadIDPath.
+	uploadIDPath := path.Join(bucket, object, uploadID)
+
+	// Cleanup uploadID for all disks.
+	for index, disk := range storageDisks {
+		if disk == nil {
+			errs[index] = traceError(errDiskNotFound)
+			continue
+		}
+		wg.Add(1)
+		// Cleanup each uploadID in a routine.
+		go func(index int, disk StorageAPI) {
+			defer wg.Done()
+			err := cleanupDir(disk, minioMetaMultipartBucket, uploadIDPath)
+			if err != nil && errorCause(err) != errVolumeNotFound {
```

Comment:
It is because `minioMetaMultipartBucket` might not existing but we don't need to throw an error for that and safely ignore it..  But it was an early requirement since sometime we don't delete `minioMetaMultipartBucket` ever. So this case won't happen. 

This can be safely removed.